sdk/eventhub/azure-eventhubs/azure/eventhub/_consumer_client.py

```diff
@@ -172,13 +175,3 @@ def close(self):
                 _, ep = self._event_processors.popitem()
                 ep.stop()
             super(EventHubConsumerClient, self).close()
-
-
-class Task:
-    def __init__(self, event_processor):
-        self._event_processor = event_processor
-
-    def cancel(self):
-        # pylint: disable=protected-access
-        EventHubConsumerClient._stop_eventprocessor(self._event_processor)
-        log.info("Task of EventProcessor %r has been cancelled successfully.", self._event_processor._id)
```

Comment:
If we remove this logic - what is the new path for a user shutting down a specific processor?

Reply:
user can't stop a specific processor. If he wants to stop, he can only stop the client.
As what we've learned in the past few month, the most common usage scenario for EventHub is using EventProcessor/EventHubProcessor to reads events infinity from every partition. So we decided not to have the ability to stop a single event processor in a producer client as that's not the common case.